src/rules/constructor_super.rs

```diff
@@ -32,6 +32,51 @@ impl LintRule for ConstructorSuper {
     let mut visitor = ConstructorSuperVisitor::new(context);
     visitor.visit_module(module, module);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Verifies the correct usage of constructors and calls to super().
+
+Defined constructors of derived classes (e.g. class A extends B) must always call
+super().  Classes which extend non-constructors (e.g. class A extends null) must
+not have a constructor.
```

Comment:
```suggestion
  fn docs(&self) -> &'static str {
    r#"Verifies the correct usage of constructors and calls to `super()`.

Defined constructors of derived classes (e.g. `class A extends B`) must always call
`super()`.  Classes which extend non-constructors (e.g. `class A extends null`) must
not have a constructor.
```

I think it would be nice to get inline codes to be wrapped in backticks